src/SprykerEco/Shared/Adyen/AdyenConstants.php

```diff
@@ -21,6 +21,8 @@ interface AdyenConstants
     public const PAY_PAL_RETURN_URL = 'ADYEN:PAY_PAL_RETURN_URL';
     public const ALI_PAY_RETURN_URL = 'ADYEN:ALI_PAY_RETURN_URL';
     public const WE_CHAT_PAY_RETURN_URL = 'ADYEN:WE_CHAT_PAY_RETURN_URL';
+    public const KLARNA_INVOICE_RETURN_URL = 'ADYEN:KLARNA_INVOICE_RETURN_URL';
```

Comment:
These constants are public api and should have api tag with specification.

Reply:
Added specification and API tag to all public constants in this class